### PR TITLE
SK-1297: getSignedDidCreationExtrinsic

### PR DESCRIPTION
**Pull request by lukeg90**
[SK-1297](https://kiltprotocol.atlassian.net/browse/SK-1297)

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -103,22 +119,38 @@ The extension MUST only inject itself into pages having the `window.kilt` object
...
 * The type ecdsa uses ECDSA signature scheme with secp256k1 curve keypair and blake2 as the hashing algorithm.
 
 
+## DID Creation
+
+1. The dApp calls `getSignedDidCreationExtinsic` passing the submitter as a parameter.
+2. The extension presents an interface for generating and signing the on-chain DID creation extrinsic.
+    It SHOULD allow the user to choose the keypair for signing.
+    It SHOULD allow the user to reject the creation and signing of the extrinsic.
+3. The extension generates and signs the on-chain DID creation extrinsic
```
Comment (arty-name) [inline readme.md:147]:
```suggestion
3. The extension generates and signs the on-chain DID creation extrinsic.
```

File: readme.md [hunk 2]
```diff
@@ -103,22 +119,38 @@ The extension MUST only inject itself into pages having the `window.kilt` object
...
+    It MUST include an authentication key.
+    It MAY include other keys and services.
+4. The extension resolves the promise returned from `getSignedDidCreationExtinsic` with the signed extrinsic.
+5. The dApp submits the extrinsic or sends it to the the submitter it specified in the API call.
```
Comment (arty-name) [inline readme.md:151]: I’d rather not include this point at all. This is already beyond the scope of the API we’re describing.

**Review by arty-name — APPROVED** [review]

File: readme.md (hunk 2 above)
Comment (arty-name) [reply readme.md:151]: I meant point 5:
>5. The dApp submits the extrinsic or sends it to the the submitter it specified in the API call.

**Review by tjwelde — APPROVED** [review]
LGTM

**Pull request merged**